tests/scripts/check-files.py

```diff
@@ -132,6 +133,16 @@ def issue_with_line(self, line):
         return b"\t" in line
 
 
+class MergeArtifactIssueTracker(IssueTracker):
+
+    def __init__(self):
+        super().__init__()
+        self.heading = "Merge artifact:"
+
+    def issue_with_line(self, line):
+        return re.match(br'\A(<<<<<<<|\|\|\|\|\|\|\||=======|>>>>>>>) ', line)
```

Comment:
Why does the merge marker have to be followed by a space?

Reply:
I want to avoid false positives. In particular for `=======` it can be a markdown title marker, or a highly visible separator, so `\A=======` would have false positives. I'm not very familiar with git configurations, when would the merge marker not be followed by a space?

Reply:
This doesn't work I'm afraid. `=======` isn't followed by a space as a conflict marker, all the rest are. You'll need a `$` after `=======` rather than a space.

Reply:
Ah, damn. A line `=======` is legitimate in markdown files at least. These sanity checks can't be perfect, so what do you prefer? Don't check `=======`, or check it only in non-markdown files?

Reply:
I think excluding markdown would be the better approach, at least some coverage is happening then.

Reply:
But what about `=====` conflict markers in markdown? or what if the markdown is documenting what conflict markers look like? :p

Reply:
Or what if there is Markdown-like title underlining in `tests/scripts/generate_test_code.py`? It's a heuristic. There is no done, there is only good enough.